akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -55,7 +58,7 @@ abstract class GraphStage[S <: Shape] extends Graph[S, Unit] {
  *    of the enclosing [[GraphStage]]
  *  * Possible mutable state, accessible from the [[InHandler]] and [[OutHandler]] callbacks, but not from anywhere
  *    else (as such access would not be thread-safe)
- *  * The lifecycle hooks [[preStart()]] and [[postStop()]]
+ *  * The lifecycle hooks preStart() and postStop()
```

Comment:
it's not a problem with such links in general, those should work as [[#preStart]] and [[#postStop]]
Otherwise backtick